server/lib/chat/chat-models.ts

```diff
@@ -40,7 +40,7 @@ function convertUserChannelEntryFromDb(props: DbUserChannelEntry): UserChannelEn
  * Gets a user channel entry for each channel that a particular user is in, ordered by their channel
  * join date.
  */
-export async function getChannelsForUser(userId: SbUserId): Promise<UserChannelEntry[]> {
+export async function getUserChannelEntriesForUser(userId: SbUserId): Promise<UserChannelEntry[]> {
```

Comment:
I think the previous naming was better, I don't really get why you handed this name to the function you created